docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md

```diff
@@ -1,48 +1,52 @@
 
 > [!NOTE]
 > Because COM and VSTO add-ins can only be installed on Windows, this article refers to COM and VSTO add-ins collectively as "Windows-only" add-ins.
 
-[!INCLUDE [new-outlook-vsto-com-support](../includes/new-outlook-vsto-com-support.md)]
-
-You can configure your Office Add-in so that when the equivalent Windows-only add-in is already installed on a user's computer, Office on Windows runs the Windows-only add-in instead of the Office Add-in. The Windows-only add-in is called "equivalent" because Office will seamlessly transition between the Windows-only add-in and the Office Add-in according to which one is installed on a user's computer. For more information, see [Equivalent behavior for users](#equivalent-behavior-for-users). As a general rule, add-ins should be treated as equivalent only when the following conditions are met, but there may be exceptional scenarios where add-ins are that don't meet these conditions can be treated as equivalent.
+ As a general rule, add-ins should be treated as equivalent only when the following conditions are met, but there may be exceptional scenarios where add-ins are that don't meet these conditions can be treated as equivalent.
 
 - They're both created by the same developer, typically the entity specified in the [`"developer"`](/microsoft-365/extensibility/schema/root-developer) property of the unified manifest or the [ProviderName](/javascript/api/manifest/providername) element in the add-in only manifest.
 - They both are designed to be installed on the same Office application or applications, and they address the same workload with mainly overlapping functionality.
 - They have identical, or highly similar, public names, and user interfaces including control names and icons.
 
-If the Windows-only add-in is ever uninstalled, Office will automatically activate the Office Add-in the next time the host Office application is started.
 
 [!INCLUDE [Support note for equivalent add-ins feature](../includes/equivalent-add-in-support-note.md)]
 
-> [!TIP]
-> Currently, the Windows-only add-in is always given preference to the Office Add-in. But there's an enhancement to this feature in preview that provides you with a way to reverse this preference or to give the user a choice between the Windows-only add-in and Office Add-in. For more information, see [Option to disable the Windows-only add-in instead (preview)](#option-to-disable-the-windows-only-add-in-instead-preview).
+[!INCLUDE [new-outlook-vsto-com-support](../includes/new-outlook-vsto-com-support.md)]
+
+Because the Windows-only and the Office Add-in may not have exactly the same features, there are three scenarios you need to consider.
```

Comment:
```suggestion
Because the Windows-only add-in and the Office Add-in may not have exactly the same features, there are three scenarios you need to consider.
```